client/app/scripts/liveblog-edit/module.js

```diff
@@ -232,8 +231,8 @@ define([
             },
             buildOwner: function(userID) {
                 api('users').getById(userID).then(function(data) {
-                    vm.original_creator = data;
-                    vm.lastOwnerId = userID;
+                    vm.temp_chosen = vm.original_creator = data;
```

Comment:
Do you really need `temp_chosen` ?
If yes, can you explain here with a comment in the code his role and rename it with a meaningful name please ? :)